projects/ndpi/build.sh

```diff
@@ -112,8 +112,8 @@ mkdir sydr && cd sydr
 $CC $CFLAGS -I $INCLUDE_DIR -I /nDPI/example /nDPI/load_sydr_ndpi_reader.c /nDPI/fuzz/fuzz_common_code.c /nDPI/example/reader_util.c $LIB_DIR/libndpi.a /libpcap-1.9.1/libpcap.a /json-c/sydr/libjson-c.a -lm -o load_sydr_ndpi_reader
```

Comment:
`fuzz_ndpi_reader` and `fuzz_process_packet` get updated in repo, while corresponding Sydr wrappers stay unchanged.
Why don't build them the same way like `quic_get_crypto` and `tls_certificate` - without separate `load_sydr_*` file?